app/src/androidTest/java/com/android/sample/ui/activity/EditActivityScreenTest.kt

```diff
@@ -0,0 +1,105 @@
+    navigationActions = mockk(relaxed = true)
+    listActivitiesViewModel = ListActivitiesViewModel(activitiesRepository)
+    listActivitiesViewModel.selectActivity(activity)
+
+    `when`(navigationActions.currentRoute()).thenReturn(Screen.EDIT_ACTIVITY)
+  }
+
+  @Test
+  fun displayAllComponents() {
+    listActivitiesViewModel.selectActivity(activity)
+    composeTestRule.setContent { EditActivityScreen(listActivitiesViewModel, navigationActions) }
+
+    composeTestRule.onNodeWithTag("editScreen").assertIsDisplayed()
+    composeTestRule.onNodeWithTag("editActivityTitle").assertIsDisplayed()
+    composeTestRule.onNodeWithTag("editActivityTitle").assertTextEquals("Edit Task")
+    composeTestRule.onNodeWithTag("goBackButton").assertIsDisplayed()
+    composeTestRule.onNodeWithTag("activitySave").assertIsDisplayed()
+    composeTestRule.onNodeWithTag("activitySave").assertTextEquals("Save")
+    composeTestRule.onNodeWithTag("activityDelete").assertIsDisplayed()
+    composeTestRule.onNodeWithTag("activityDelete").assertTextEquals("Delete")
+
+    composeTestRule.onNodeWithTag("inputTitleEdit").assertIsDisplayed()
+    composeTestRule.onNodeWithTag("inputDescriptionEdit").assertIsDisplayed()
+    composeTestRule.onNodeWithTag("inputAssigneeEdit").assertIsDisplayed()
+    composeTestRule.onNodeWithTag("inputLocationEdit").assertIsDisplayed()
+    composeTestRule.onNodeWithTag("inputDateEdit").assertIsDisplayed()
+    composeTestRule.onNodeWithTag("inputStatusEdit").assertIsDisplayed()
+  }
+
+  @Test
+  fun inputsHaveInitialValue() {
+    listActivitiesViewModel.selectActivity(activity)
+    composeTestRule.setContent { EditActivityScreen(listActivitiesViewModel, navigationActions) }
+
+    composeTestRule.waitForIdle()
+
+    composeTestRule.onNodeWithTag("inputTitleEdit").assertTextContains(activity.title)
+    composeTestRule.onNodeWithTag("inputDescriptionEdit").assertTextContains(activity.description)
+    composeTestRule.onNodeWithTag("inputAssigneeEdit").assertTextContains(activity.creator)
+    composeTestRule.onNodeWithTag("inputDateEdit").assertTextContains("5/9/2024")
+    composeTestRule.onNodeWithTag("inputStatusEdit").assertTextContains("Active")
+  }
+
+  @Test
+  fun correctlyIteratesStatuses() {
+    listActivitiesViewModel.selectActivity(activity)
+    composeTestRule.setContent { EditActivityScreen(listActivitiesViewModel, navigationActions) }
+    composeTestRule.onNodeWithTag("inputStatusEdit").assertTextContains("Active")
+
+    composeTestRule.onNodeWithTag("inputStatusEdit").performClick()
+    composeTestRule.onNodeWithTag("inputStatusEdit").assertTextContains("Started")
+
+    composeTestRule.onNodeWithTag("inputStatusEdit").performClick()
+    composeTestRule.onNodeWithTag("inputStatusEdit").assertTextContains("Ended")
+
+    composeTestRule.onNodeWithTag("inputStatusEdit").performClick()
+    composeTestRule.onNodeWithTag("inputStatusEdit").assertTextContains("Archived")
+  }
+}*/
```

Comment:
all is commented. Is it an expected thing ? I'd recommend to simply remove the file, and keep it locally until needed. If not expected, make sure to uncomment them.